miner/main/index.ts

```diff
@@ -53,21 +55,24 @@ export default class Miner {
   private readonly httpRoutes: [url: RegExp | string, method: string, handler: IHttpHandleFn][];
   private readonly wsRoutes: [RegExp | string, IWsHandleFn][] = [];
 
-  constructor(addressHost = 'localhost') {
+  constructor(addressHost = 'localhost', shouldShutdownOnSignals = true) {
```

Comment:
It seems the second argument should be an options object, same as Hero. I'm going to go ahead and merge though.